lightly/api/api_workflow_predictions.py

```diff
@@ -137,17 +137,19 @@ def create_or_update_prediction(
         prediction_singletons: Sequence[PredictionSingletonRepr],
         prediction_version_id: int = -1,
     ) -> None:
-        """Creates or updates the predictions for one specific sample
+        """Creates or updates predictions for one specific sample.
 
         Args:
             sample_id
-                The id of the sample
+                The ID of the sample.
 
             prediction_version_id:
-                And id to distinguish different predictions for the same sample.
+                A numerical ID (e.g timestamp) to distinguish different predictions of different model versions.
```

Comment:
Please update the `(e.g timestamp)` part to match the docstring from further above.